assets/templates/ViewDonars.html

```diff
@@ -32,6 +33,7 @@ <h2 class="text-white mb-4">View Donars</h2>
                     <td>{{v.fullname}}</td>
                     <td>{{v.email}}</td>
                     <td>{{v.fooditems}}</td>
+                    <td><img   width="200px" height="250px" src = "../media/img.jpg"></td>
```

Comment:
When I try to add a new image for a new food item from donor side, I am not able to see the new image but it is displaying the old image. Can you please look into it?

Reply:
Yup I have made the necessary changes once please look into if it's ok I will merge that.

Reply:
It's working. Go ahead and merge it.